### DISCUSSION TITLE
0.9.4

### DISCUSSION THREAD

**Discussion by GhostofGoes** [GhostofGoes/getmac#88] [General]
### Added
* Support BusyBox's `arping`

### Changed
* Improve how ARP is handled. If `ArpFile` method succeeds, use it instead of `ArpingHost` (this should fix [#86](https://github.com/GhostofGoes/getmac/issues/86), for realsies this time).
* Speed up the first call to `ArpingHost`
* Fix FORCE_METHOD not being respected for IPv4 macs

<hr /><em>This discussion was created from the release <a href='https://github.com/GhostofGoes/getmac/releases/tag/0.9.4'>0.9.4</a>.</em>

**Comment by ReimundR** [comment]
Hi @GhostofGoes , will there be a 0.9.5 for the macOS fix #93 ?

**Reply by GhostofGoes** [reply]
Yes, apologies for the delay, life and work has been pulling me away a lot this year. I'll try to get it out in the next week.

**Reply by ReimundR** [reply]
Is no problem, both have priority. It's voluntary work here. Thank you for that )
